android/titanium/src/java/org/appcelerator/titanium/view/TiUIView.java

```diff
@@ -831,19 +831,6 @@ public void propertyChanged(String key, Object oldValue, Object newValue, KrollP
 					background.setCallback(null);
 					background = null;
 				}
-
-				if (d.containsKeyAndNotNull(TiC.PROPERTY_BACKGROUND_COLOR)) {
-					Integer bgColor = TiConvert.toColor(d, TiC.PROPERTY_BACKGROUND_COLOR);
-					if (!nativeViewNull) {
-						if (canApplyTouchFeedback(d)) {
-							applyTouchFeedback(bgColor, d.containsKey(TiC.PROPERTY_TOUCH_FEEDBACK_COLOR)
-															? TiConvert.toColor(d, TiC.PROPERTY_TOUCH_FEEDBACK_COLOR)
-															: null);
-						} else {
-							nativeView.setBackgroundColor(bgColor);
-						}
-					}
-				}
```

Comment:
I think removing this block of code will cause a regression where you can no longer change a view's `backgroundColor` dynamically anymore because `setBackgroundColor()` won't get called.